### PR TITLE
[docs] Lint all legacy docs for spelling, attributes, repeated words

### PR DESCRIPTION
**Pull request by colleenmcginnis**
## Motivation/summary

This PR lints all `*.asciidoc` files in the `docs/legacy/` directory. Lints for:
- Spelling
- Shared attribute use
- Repeated words

Note: I made edits to `docs/legacy/fields.asciidoc` in a separate commit because it is soooo long and I wasn't sure if it was generated and should be edited elsewhere (then we can drop the commit here).

## Related issues

https://github.com/elastic/apm-server/pull/8164

### CODE REVIEW

File: docs/legacy/breaking-changes.asciidoc [hunk 1]
```diff
@@ -31,7 +31,7 @@ There are three breaking changes to be aware of;
 these changes only impact users ingesting data with
 {apm-server-ref-v}/jaeger.html[Jaeger clients].
 
-* Leading 0s are no longer removed from Jaeger client trace/span ids.
+* Leading 0 seconds are no longer removed from Jaeger client trace/span ids.
```
Comment (bmorelli25) [inline docs/legacy/breaking-changes.asciidoc:34]:
```suggestion
* Leading zeros are no longer removed from Jaeger client trace/span ids.
```

File: docs/legacy/breaking-changes.asciidoc [hunk 2]
```diff
@@ -87,16 +87,16 @@ There are no breaking changes in APM Server.
...
-As a side effect, `setup.template.*` settings will be ignored for ILM related templates per event type.
-See {apm-server-ref}/ilm.html[set up ILM] for more information.
+This means you can now customize {ilm-init} behavior from within the APM Server configuration.
+As a side effect, `setup.template.*` settings will be ignored for {ilm-init} related templates per event type.
+See {apm-server-ref}/ilm.html[set up {ilm-init}] for more information.
 
-* By default, ILM policies will not longer be versioned.
-All event types will switch to the new default policy: rollover after 30 days or when reaching a size 50gb.
+* By default, {ilm-init} policies will not longer be versioned.
+All event types will switch to the new default policy: rollover after 30 days or when reaching a size 50 GB.
```
Comment (bmorelli25) [inline docs/legacy/breaking-changes.asciidoc:95]:
```suggestion
All event types will switch to the new default policy: rollover after 30 days or when reaching a size of 50 GB.
```

File: docs/legacy/configuration-process.asciidoc [hunk 3]
```diff
@@ -40,7 +40,7 @@ output.elasticsearch:
 [float]
 ==== `host`
 Defines the host and port the server is listening on.
-Use "unix:/path/to.sock" to listen on a unix domain socket.
+Use `unix:/path/to.sock` to listen on a Unix domain socket.
```
Comment (bmorelli25) [inline docs/legacy/configuration-process.asciidoc:43]:
```suggestion
Use `"unix:/path/to.sock"` to listen on a Unix domain socket.
```

**Review by bmorelli25 — COMMENTED** [review]
Just a few suggestions.

File: docs/legacy/breaking-changes.asciidoc (hunk 1 above)
Comment (colleenmcginnis) [reply docs/legacy/breaking-changes.asciidoc:34]: Oh wow. 🙈

**Review by bmorelli25 — APPROVED** [review]
🚢

**Pull request merged**